issuer/upload-claims/scripts/deploy.js

```diff
@@ -14,26 +14,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const { ethers, upgrades } = require('hardhat');
-const os = require('os');
-const fs = require('fs');
-const path = require('path');
-const pathOutputJson = path.join(os.homedir(), './iden3/deploy_output.json');
+const { ethers, upgrades } = require("hardhat");
+const os = require("os");
+const fs = require("fs");
+const path = require("path");
+const workDir = process.env.IDEN3_WORKDIR || os.homedir();
```

Comment:
Suggestion: don't require `iden3` as top-level directory, i.e. let the default be `IDEN3_WORKDIR = "~/iden3"` instead, but allow user to override.

Reply:
@nedgar sounds great